compiler/rustc_resolve/src/lib.rs

```diff
@@ -2177,43 +2209,45 @@ impl<'ra, 'tcx> Resolver<'ra, 'tcx> {
         }
     }
 
-    fn extern_prelude_get(&mut self, ident: Ident, finalize: bool) -> Option<NameBinding<'ra>> {
-        let mut record_use = None;
-        let entry = self.extern_prelude.get(&ident.normalize_to_macros_2_0());
-        let binding = entry.and_then(|entry| match entry.binding.get() {
-            Some(binding) if binding.is_import() => {
-                if finalize {
-                    record_use = Some(binding);
-                }
-                Some(binding)
-            }
-            Some(binding) => {
+    fn extern_prelude_get<'r>(
```

Comment:
Could you rebase this on master properly?
This function was rewritten, but this PR contains the old version.